Stripe/STPBancontactSourceInfoDataSource.m

```diff
@@ -10,13 +10,14 @@
 
 #import "NSArray+Stripe_BoundSafe.h"
 #import "STPLocalizationUtils.h"
+#import "STPTextFieldTableViewCell.h"
 
 @implementation STPBancontactSourceInfoDataSource
 
 - (instancetype)initWithSourceParams:(STPSourceParams *)sourceParams {
     self = [super initWithSourceParams:sourceParams];
     if (self) {
-        self.title = STPLocalizedString(@"Bancontact Info", @"Title for form to collect Bancontact account info");
+        self.paymentMethodLabel = STPLocalizedString(@"Bancontact", @"Text for Bancontact payment method");
```

Comment:
Should we align with STPPaymentMethodType in these classes somehow? It has some of the UI image/text bits in it already, and i assume support for those types and these data sources will be 1:1